.dockerignore

```diff
@@ -12,3 +12,4 @@ venv/
 /.github/
 /netlify.toml
 /setup/
+.github/
```

Comment:
Accidental inclusion? :smile:

Reply:
nope, .github is not needed inside container

Reply:
No worries. :smile: